serverless.js

```diff
@@ -221,21 +224,23 @@ module.exports = {
                 },
               },
               {
-                DomainName: {
-                  'Fn::GetAtt': ['StorybookBucket', 'RegionalDomainName'],
+                CustomOriginConfig: {
+                  OriginProtocolPolicy: 'http-only',
                 },
-                Id: 's3origin-storybook',
-                S3OriginConfig: {
-                  OriginAccessIdentity: {
-                    'Fn::Join': [
-                      '/',
-                      [
-                        'origin-access-identity/cloudfront',
-                        { Ref: 'CloudFrontOriginAccessIdentityStorybook' },
+                DomainName: {
+                  'Fn::Select': [
+                    '1',
+                    {
+                      'Fn::Split': [
+                        'http://',
+                        {
+                          'Fn::GetAtt': ['StorybookBucket', 'WebsiteURL'],
```

Comment:
We are accessing the bucket directly 😐
That explains having the http configuration.

Reply:
Yes, exactly. As stated in my comments this is the same setup as https://aws.amazon.com/premiumsupport/knowledge-center/cloudfront-serve-static-website/